playground/infrastructure/grpc_client.py

```diff
@@ -130,6 +130,21 @@ async def get_compile_output(self, pipeline_uuid: str) -> str:
     response = await self._stub.GetCompileOutput(request)
     return response.output
 
+  async def get_graph(self, pipeline_uuid: str) -> str:
```

Comment:
We have graph only for Java and Python SDKs, do we cover it somehow?

Reply:
In this PR no. I think I can add check into the CI/CD and ask for a graph only if it is java/python SDK.

Reply:
Added